### DISCUSSION TITLE
Python Hatchery Participation

### DISCUSSION THREAD

**Discussion by VeldaKiara** [BlackPythonDevs/blackpythondevs#24] [General]
Jay's Suggestions

> I was thinking about a Black Python Panel but this is a full day event and I'm not sure how we'd fill an entire day. 
> 
> The biggest thing that we can't (and shouldn't) do is create a space where black speakers go to give their talks because that creates a problem where it looks like we're segregating ourselves.
> 
> I don't know if the hatchery allows for things like fundraisers. Ultimately I would love for folks to share some ideas that they'd like to see. Even if it doesn't become a part of the Hatchery, I could see some of the ideas going into a Black Python Devs event.

Velda's Suggestions

> I have a suggestion since we eliminated the bit about presenting talks to avoid segregation, what if we have an open conversation about what open source projects people are working on the issues they have solved what's left and how we can help. Also, I want to believe we have maintainers too who would love a forum to talk about the projects they maintain, offer tips and what they would equally love to get from the community
> I don't know if this still happens in PyconUs, let me know what you think and what your thoughts are for the hatchery
> Also, we could have time for people to contribute to BPD's website and design too

Jay's suggestions

> I like this! I'm thinking perhaps we can get some folks that are running some larger projects and see if they would be willing to present on a panel talking about the things that they need. I can assume that David Lord from Flask (Pallets), Sebastian from FastAPI and someone that is a part of the DSF will be there.

> I think there is an idea of like chatting with maybe Dawn or someone from Djangonaut space to have them chat about their successes of getting more diverse contributors with a scheduled sprints. Or maybe even like trying to host a sprint room or something.
> It would be welcome to everyone
> I think the problem with that is when, because doing it during the conference means people have to choose and then getting people to commit to coming earlier/later won't be something we can rely on 

Laz's suggestions
I like that idea of presenting and showing the benefits of the Djangonaut model. I like that it can extend to something that continues passed the end of  PyCon. Though the commitment issue can still be there. 

We talked about at pycon last year @kjaymiller,  that everyone is going to want something different from  Black Python Devs community.  The hatchery can really help us come out of the gates swinging. So I encourage anyone reading this  to take the opportunity to let your ideas be known

Potential Needs:
- [ ] Placards
- [ ] Signs/Sign Boards

**Comment by VeldaKiara** [comment]
This is where the discussion is at, to this point.

**Reply by kjaymiller** [reply]
@VeldaKiara since you are taking the lead on this I would love for you to steer the convo a bit on this. If you need me to send emails or get in touch with potential guests I can do that.

I like the idea of a contributors open house. I think this would be a good event to try and do during the workshops the day before. We may be able to make this happen and I would expect this would be around the time that a lot of folks are going to check in and be around the venue anyway. So perhaps it could be a setup tables with little signs and have folks sign up for a spot and then people can drop in and talk to maintainers about the different types of contributions that people are looking for and whether or not the maintainers are staying for sprints.

This is a "Helps Everyone" type thing but I think it would allow us to fundraise and also we can make it so that Black Python Devs members have priority registration or something to that effect

**Reply by VeldaKiara** [reply]
I don't know a lot of people, so helping with the guests will be a huge plus.

**Reply by VeldaKiara** [reply]
So we might need placards and sign boards to show this

**Reply by VeldaKiara** [reply]
How do we identify the members?

**Reply by kjaymiller** [reply]
I think we just reach open the space to BPD members early. Depending on the space, there will be limited space so we'll likely need to have folks sign up and it will be first come first serve.

**Comment by kjaymiller** [comment]
> I don't know a lot of people, so helping with the guests will be a huge plus.

I think we can reach out to some folks informally and get their interest in participating. then we can also look at the PSF to tell us about how much space we would have to then take requests for other participants
